### PR TITLE
CALCOM-7895 - [CAL-1320] remove "application_fee_amount" for all regions

### LINKED ISSUE #7895: [CAL-1320] remove "application_fee_amount" for some regions (India, Mexico, Brazil, ..)
Stripe connect can't charge an application fee for:

Brazil, India, Mexico, Malaysia, Singapore, Thailand

we need to exclude those regions as it makes the Stripe App unusable for customers from those regions.

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1320](https://linear.app/calcom/issue/CAL-1320/remove-application-fee-amount-for-some-regions-india-mexico-brazil)</sub>

### PR DESCRIPTION
**Pull request by gitstart-calcom**
Fixes #7895

### CODE REVIEW

File: packages/app-store/stripepayment/lib/PaymentService.ts [hunk 1]
```diff
@@ -61,7 +61,6 @@ export class PaymentService implements IAbstractPaymentService {
         amount: payment.amount,
         currency: this.credentials.default_currency,
         payment_method_types: ["card"],
-        application_fee_amount: paymentFee,
       };
```
Comment (hariombalhara) [inline packages/app-store/stripepayment/lib/PaymentService.ts:64]: Verified that collected fee doesn't show up for me with this branch. It is collected in main
Screenshot 2023-04-11 at 9 21 12 PM

Comment (hariombalhara) [reply packages/app-store/stripepayment/lib/PaymentService.ts:64]: Screenshot 2023-04-11 at 9 24 04 PM
Should we update it here as well ? cc @PeerRich I am not sure how long we would remain without fee, so it should reflect that ideally.

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Waiting on confirmation from @PeerRich. Otherwise it's good to merge. Blocking till then

File: packages/app-store/stripepayment/lib/PaymentService.ts (hunk 1 above)
Comment (PeerRich) [reply packages/app-store/stripepayment/lib/PaymentService.ts:64]: worth keeping, we will readd it later

**Review by PeerRich — APPROVED** [review]

**Review by hariombalhara — APPROVED** [review]

**Pull request merged**